src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,71 @@
-import React from 'react';
+import React, { ChangeEvent, FormEvent, useState } from 'react';
 import './FindMovie.scss';
+import cn from 'classnames';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+
+const PICTURE = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+interface Props {
+  onAddMovie: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ onAddMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isError, setIsError] = useState<boolean>(false);
+  const [isLoaded, setIsLoading] = useState<boolean>(false);
```

Comment:
TS already knows about this types, bc you put an initial value
```suggestion
  const [isError, setIsError] = useState(false);
  const [isLoaded, setIsLoading] = useState(false);
```